src/js/base/ProgressCircle.tsx

```diff
@@ -0,0 +1,223 @@
+import { Indicator, Root } from "@radix-ui/react-progress";
+import React from "react";
+import styled, { keyframes } from "styled-components";
+import { getColor, sizes } from "../app/theme";
+import { JobState } from "../jobs/types";
+
+/**
+ * Calculate the stroke width based on the total size of the progress circle
+ *
+ * @param size - The total size of the progress circle in pixels
+ * @returns  The stroke width in pixels
+ */
+
```

Comment:
Should be `calculateStrokeWidth` since `get` is pretty much reserved from getting values out of an object.

If you are going to pass a props object right to the function, the `get` naming is appropriate. I think I'd make all these functions take `props`. It will clean up your SC styling as well.

```suggestion
 */
```